### ISSUE TITLE
Segmentation fault from `interval_parameters`

### ISSUE THREAD

**Issue by dpsanders** [dpsanders/ValidatedNumerics.jl#66]
```
julia> using ValidatedNumerics

julia> ValidatedNumerics.interval_parameters
ValidatedNumerics.IntervalParameters(Float64,256,:narrow,
signal (11): Segmentation fault: 11
__gmpn_mul_basecase at /Users/dsanders/development/julia-dev/usr/lib/libgmp.10.dylib (unknown line)
Segmentation fault: 11
```

**Comment by dpsanders** [comment]
This seems to be because `pi` is left undefined in
https://github.com/dpsanders/ValidatedNumerics.jl/blob/01eb85e5dd477dec93b50ee892c1dbaee272b997/src/intervals/parameters.jl#L10

**Comment by lbenet** [comment]
This is related to #62

**Comment by lbenet** [comment]
I think we should either define it explicitly, or get rid of it. It is actually never used.

**Comment by dpsanders** [comment]
Fixed by #69 .
